libs/design-system/logos/apps/src/lib/app-logos.component.stories.ts

```diff
@@ -0,0 +1,62 @@
+import type { Meta, StoryObj } from '@storybook/angular';
+import { AppLogosComponent } from './app-logos.component';
+
+const meta: Meta<AppLogosComponent> = {
+  component: AppLogosComponent,
+  title: 'AppLogosComponent',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/BCEJn9KCIbBJ5MzqnojKQp/Design-System-Components?node-id=75-4',
+    },
+  },
+};
+export default meta;
+type Story = StoryObj<AppLogosComponent>;
+
+export const DefaultFTU: Story = {
+  args: {
+    appLink: 'https://apps.humanatlas.io/ftu-explorer/#/',
+    appIcon: 'assets/logo/ftu_logo.svg',
+    appTitle: 'FTU Explorer',
+    appDescription: 'HRA Preview Application',
+  },
+};
+
+export const DefaultCDE: Story = {
```

Comment:
I think we can consolidate these into two stories, maybe three if we can set options for the viewport plugin. Let's work together to achieve this